File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Photo.java
package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Photo {

    @Expose
    private Long height;
    @SerializedName("html_attributions")
    private List<String> htmlAttributions;
    @SerializedName("photo_reference")
    private String photoReference;
    @Expose
    private Long width;

    public Long getHeight() {
        return height;
    }

    public void setHeight(Long height) {
        this.height = height;
    }

    public List<String> getHtmlAttributions() {
        return htmlAttributions;
    }

    public void setHtmlAttributions(List<String> htmlAttributions) {
        this.htmlAttributions = htmlAttributions;
    }

    public String getPhotoReference() {
        return photoReference;
    }

    public void setPhotoReference(String photoReference) {
        this.photoReference = photoReference;
    }

    public Long getWidth() {
        return width;
    }

    public void setWidth(Long width) {
        this.width = width;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Photo.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "Photo")
data class Photo(@ColumnInfo(name = "height") var height: Long?,
                 @SerializedName("html_attributions")
                 @Expose
                 @ColumnInfo(name = "htmlAttributions") var htmlAttributions: List<String>?,
                 @SerializedName("photo_reference")
                 @Expose
                 @ColumnInfo(name = "photoReference") var photoReference: String?,
                 @ColumnInfo(name = "width") var width: Long?)